lms/djangoapps/course_api/blocks/api.py

```diff
@@ -17,7 +17,9 @@ def get_blocks(
         requested_fields=None,
         block_counts=None,
         student_view_data=None,
-        return_type='dict'
+        return_type='dict',
+        block_types=None,
+        lti_url=None
```

Comment:
Thanks @douglashall.  I still have some questions, and maybe the comment needs to be clarified in views.py, but I get that it is independent of this PR.  Thanks.  We'll meet soon.